index.html

```diff
@@ -0,0 +1,33 @@
+  <!DOCTYPE html>
+  <html lang="en">
+  <head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My portfolio</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+  <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+  <main class="about-me">
+  <nav>
+  <header class="logo">
+  <h3>Welcome</h3>
+  <a href="" class="menu"><i class="fas fa-bars"></i></a>
+  </header>
+  </nav>
+  <h1>Hey There. I'm Aster</h1>
+  <h2>I'am a Software developer</h2>
+  <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+  <section class="hero">
+  <ul class="social">
```

Comment:
- [ ] [OPTIONAL] Kindly change the color of the social icons to the original color provided by the figma template. This will make your design look more like the template provided.